specs/seo/baseUrl.spec.ts

```diff
@@ -8,8 +8,10 @@ await setup({
   // overrides
   nuxtConfig: {
     i18n: {
-      defaultLocale: 'en',
-      baseUrl: () => {
+      defaultLocale: 'en'
+    },
+    site: {
+      url: () => {
```

Comment:
@harlan-zw `nuxt-site-config` currently does not allow options to be set using functions as it is currently supported by `nuxt-i18n`. Are there plans to change this or alternatives to be used?
I'd guess this could be transformed into making use of the `site-config:resolve` hook, right?